SpatialGDK/Source/SpatialGDKEditor/Private/SchemaGenerator/SpatialGDKEditorSchemaGenerator.cpp

```diff
@@ -628,7 +628,7 @@ void ResetUsedNames()
  	}
 }
 
-void RunSchemaCompiler()
+bool RunSchemaCompiler()
```

Comment:
if `PlatformFile.CreateDirectoryTree(*SchemaDescriptorDir);` fails shouldn't this return false in that case too?

Reply:
Added